timemachine/fe/system.py

```diff
@@ -1,20 +1,60 @@
 import functools
+import multiprocessing
 
+import jax
 import jax.numpy as jnp
 import numpy as np
+import scipy
 
-from timemachine.potentials import bonded, nonbonded
+from timemachine.integrator import simulate
+from timemachine.potentials import bonded, chiral_restraints, nonbonded
+
+
+def minimize_scipy(x0, U_fn):
```

Comment:
reviewers: you can ignore this function and below function since it's just a copy and paste from #756